project/gh-action/docker/docker-compose.yml

```
@@ -0,0 +1,15 @@
+version: '2.1'  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)